### PR TITLE
Activate features

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @msarahan I never used features before.  Can you take a quick look at this PR?

### CODE REVIEW

**Comment by msarahan** [comment]
LGTM.  Assuming appveyor passes, should work fine.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,13 +12,15 @@ source:
...
-     - vc9  # [win and py27]
-     - vc10  # [win and py34]
-     - vc14  # [win and py35]
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
 
 requirements:
+    build:
+        - python  # [win]
```
Comment (pelson) [inline recipe/meta.yaml:23]: Does this not make the features redundant though?

Comment (msarahan) [inline recipe/meta.yaml:23]: No, it's different from putting Python in the runtime requirements (which is when that gets pinned.)

This is telling it to pay attention to which Python is being used to build.  Without it, I don't think the selectors are activated.  I might be wrong, though.

**Comment by ocefpaf** [comment]
Thanks @msarahan!

**Pull request merged**